libtiledbsoma/src/utils/arrow_adapter.cc

```diff
@@ -413,25 +413,25 @@ Dimension ArrowAdapter::_create_dim(
             return Dimension::create(
                 *ctx, name, type, (uint64_t*)buff, (uint64_t*)buff + 2);
```

Comment:
What about lines 414 (here) and 404 (just above) ?

Reply:
I.e. I see that they are different (Dimension objects as opposed to ArrowAdapter method calls -- but why?

Reply:
nvm. As mentioned over DM but repeating here for anyone else reviewing, the datetime types need to be explicitly passed. Replacing with `_create_dim_aux<uint64_t>` would incorrectly create these datetime dims as uint64 dims.